pywaw/pywaw/settings/local.py

```diff
@@ -1,14 +1,17 @@
 from .base import *
 
-DEBUG = True
+# we need explicit DEBUG=true
+DEBUG = get_env_var("DEBUG", default="false") == "true"
```

Comment:
I'd suggest normalizing case of DEBUG env var value - I am spoiled by YAML parsers and did not expect any difference between 'true' and 'True' ;)